products/sle/main.pm

```diff
@@ -754,6 +754,10 @@ elsif (get_var("VIRT_AUTOTEST")) {
     }
     if (get_var("VIRT_PRJ1_GUEST_INSTALL")) {
         loadtest "virt_autotest/guest_installation_run";
+        #Snapshots are supported on KVM VM Host Servers only
+        return unless (check_var('SYSTEM_ROLE', 'kvm'));
+        # VM snapshot via virsh
+        loadtest "virt_autotest/virsh_snapshot";
     }
```

Comment:
Please hold on integration of this test to prj1, since what wayne is doing now as clue scripts before calling the scripts  that you and qam newly create, is still wip.
All tasks in the backlog that qam and us share has the same position as clarified in yesterday's dns discussion meeting.